tests/integration-test/bpf/app-test.bpf.c

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+// Copyright Authors of bpfman
+
+// clang-format off
+#include <linux/bpf.h>
+#include <linux/pkt_cls.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+// clang-format on
+
+volatile const __u8 GLOBAL_u8 = 0;
+volatile const __u32 GLOBAL_u32 = 0;
+
+void print_globals(char *prefix) {
+  bpf_printk("%s: GLOBAL_u8: 0x%02X, GLOBAL_u32: 0x%08X", prefix, GLOBAL_u8,
```

Comment:
why u bother with those 2 globals if there is no userspace to set them ?

Reply:
We set the globals in integration tests and in sample programs.  That way we can `cat /sys/kernel/debug/tracing/trace_pipe` to confirm they are working as expected.

Reply:
I see the PR showed only kernel side so I didn't realize userspace already populate them

Reply:
I'm using these bpf functions in the sample programs in my load/attach split pr which hasn't merged yet.  At some point, we'll probably create a bpfman integration test that uses them all and an example program/k8s-integration test that uses them all.  However, for now, I just needed something to test all the program types in my draft pr.